proj/src/main/webapp/chat.js

```diff
@@ -71,24 +79,43 @@ function createNewChatWithUser(tag){
     var currentReference = firebase.database().ref("/chat/" + tag);
     var postKey = currentReference.push(newChat).key;
     currentReference = firebase.database().ref("/chat/" + tag + "/" + postKey + "/users/");
-    addUserToTag(currentReference, tag);
+    addUserToTag(currentReference, tag, lat, long);
 
     return postKey;
 }
 
 // Loops through open chat rooms, adds the user to the first open chat and returns the key
-function findChatAndAddUser(snapshot, tag){
+function findChatAndAddUser(snapshot, tag, lat, long){
     var key;
+    var chatLat;
+    var chatLong;
+    const precision = .5;
```

Comment:
can this precision be moved to a global constant maybe in the location.js file.

Reply:
Yes I'll do that in my next set of features